src/test/resources/test_scripts/parameterPipeline-html-in-descriptions.groovy

```diff
@@ -0,0 +1,31 @@
+node {
+
+//
+// Commented out for now because pipeline doesn't accept Job parameters
+// on the first run of the Job, which seems a bit messed up. The actual
+// components for processing the inputs are the same in both cases, so
+// testing the inputs should be fine for now anyway.
```

Comment:
Sorry, I think I follow you now ... you're not asking "why not" ... more of a "you know you could try ...".

Anyway ... from what I can see, it would require me running the job twice because the input job parameters (not input step) do not seem to appear on the first run i.e. it needs to run it ... then run it again ... i.e. a pain in the arse :)